dt/uni01alpha/networker/kustomization.yaml

```diff
@@ -0,0 +1,126 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  - |-
+    apiVersion: builtin
+    kind: NamespaceTransformer
+    metadata:
+      name: _ignored_
+      namespace: openstack
+    setRoleBindingSubjects: none
+    unsetOnly: true
+    fieldSpecs:
+      - path: metadata/name
+        kind: Namespace
+        create: true
+
+resources:
+  - ../../../lib/dataplane/dataplane-ssh-secret.yaml
+  - ../../../lib/dataplane/openstackdataplanenodeset.yaml
+  - ../../../lib/dataplane/openstackdataplanedeployment.yaml
+
+patches:
+  - target:
+      kind: OpenStackDataPlaneDeployment
+      name: .*
+    patch: |-
+      - op: replace
+        path: metadata/name
```

Comment:
should be /metadata/name

Reply:
Thank you. Will fix it in the next commit.